llvm/test/CodeGen/ARM/useaa.ll

```diff
@@ -1,4 +1,5 @@
 ; RUN: llc < %s -mtriple=armv8r-eabi -mcpu=cortex-r52 | FileCheck %s --check-prefix=CHECK --check-prefix=USEAA
+; RUN: llc < %s -mtriple=armv8r-eabi -mcpu=cortex-r52plus | FileCheck %s --check-prefix=CHECK --check-prefix=USEAA
```

Comment:
Adding these run lines for some of these tests may not be necessary for both r52 and r52+. I think I would drop it from the scheduling/useaa tests if it just uses the same scheduling model.